src/USER-CGDNA/bond_oxdna_fene.cpp

```diff
@@ -328,29 +328,9 @@ void BondOxdnaFene::coeff(int narg, char **arg)
 
 void BondOxdnaFene::init_style()
 {
```

Comment:
@ohenrich the code I have removed here is a bad idea, since changing the factors does not update the corresponding data structures and lists (you need to create and instance of Special and then call the build() method). Instead the cleaner solution is a hard error, best with the exact command line that is missing in the input deck, so that people can just cut-n-paste it (as done here). Also the fprintf() was printing on every MPI rank, which would be messy for many MPI ranks.